Calculate/CalculatorOperations.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Numerics;
+
+namespace Calculate;
+
+public static class CalculatorOperations
```

Comment:
I wonder if we need this extra class or if putting this in the Calculator class itself makes more sense?